app/client/survey_admin/survey_admin_users.coffee

```diff
@@ -0,0 +1,26 @@
+Template.survey_admin_users.onCreated ->
+  @users = new Meteor.Collection(null)
+  @fetched = new ReactiveVar false
+Template.survey_admin_users.onRendered ->
+  @survey = @data.survey
+  @users.find().map (user)=>
+    @users.remove user
+  @fetched.set(false)
+  @survey.relation('invitedUsers').query().find()
+    .then (result) =>
+      result.forEach (item)=>
+        console.log @users
+        console.log item.toJSON()
+        @users.insert item.toJSON()
+      console.log result
+    .fail (err)->
+      console.error(err)
+      toastr.error err.message
+    .always =>
+      @fetched.set(true)
+
+Template.survey_admin_users.helpers
+  fetched: ->
+    Template.instance().fetched.get()
```

Comment:
This helper shouldn't be necessary due to the globally registered helper here: 
https://github.com/ecohealthalliance/mobile-survey/blob/add-survey-users/app/client/helpers.coffee